samples/build-duration-logger/README.md

```diff
@@ -12,10 +12,18 @@ In this JavaScript example, build durations are calculated and written out to th
 
 To run this sample:
```

Comment:
I'd also add a section what steps are necessary to transfer this script to a plain HTML page and how to include the polyfill there.

Reply:
done